_website/OldCode/Basic/2-sample/chi.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+output:
+  html_document:
+    css: custom-blockquote.css
+---
+
+```{r setup&variables, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+# title: comparing two variables - chi square and Fisher's exact test 
+# author: jonathan
+# revised: 14/1/2020 
+
+WEB <- c("https://bioceed.uib.no/dropfolder/bioSTATS/Rmd/Basic/2-sample/chi.html")
+GIT <- c("https://github.com/jonathansoule/biostats/blob/master/Basic/2-sample/chi.Rmd")
+library(ggplot2)
+rain<-c(38.2,171.1,83.2,94.8,107.2,87.6,116.0,253.0,262.6,99.8,189.0,93.8)
+temperature<-c(5.6,8.1,10.7,13.6,15.8,12.9,9.4,6.6,5.5,0.6,1.8,4.3)
+df <- data.frame(rain, temperature)
+```
+
+The **Chi-square test** for independence (a.k.a. Ï-square test or Pearsonâs chi-square test of association) comes in handy when you need to compare two **categorical** variables and when the dataset is made of **counts**. Often this dataset will look like a âcontingency tableâ, something like this:
```

Comment:
special character may be problematic - try $\chi^2$. Don't think you need to mention Pearson